rbx_binary/FORMAT.md

```diff
@@ -53,71 +53,19 @@ This document is based on:
 	5. One `END` chunk
 
 ## File Header
-Every file starts with:
-
-<table>
-	<tr>
-		<th width="40">0</th>
-		<th width="40">1</th>
-		<th width="40">2</th>
-		<th width="40">3</th>
-		<th width="40">4</th>
-		<th width="40">5</th>
-		<th width="40">6</th>
-		<th width="40">7</th>
-		<th width="40">8</th>
-		<th width="40">9</th>
-		<th width="40">10</th>
-		<th width="40">11</th>
-		<th width="40">12</th>
-		<th width="40">13</th>
-		<th width="40">14</th>
-		<th width="40">15</th>
-	</tr>
-	<tr>
-		<td colspan="8">Magic number (<code>&lt;roblox!</code>)</td>
-		<td colspan="6">Signature (<code>89 ff 0d 0a 1a 0a</code>)</td>
-		<td colspan="2">Version (<code>0</code>)</td>
-	</tr>
-	<tr>
-		<td colspan="4">Number of instance types (<code>u32</code>)</td>
-		<td colspan="4">Number of instances (<code>u32</code>)</td>
-		<td colspan="8">Reserved bytes (always 0)</td>
-	</tr>
-</table>
+Every file starts with a 16 byte header.
```

Comment:
`Number of instance types`, `Number of Instances` and `Reserved bytes` appear to have disappeared!

Reply:
oops